### ISSUE TITLE
related Google project: Kythe

### ISSUE THREAD

**Issue by jokeyrhyme** [sourcegraph/srclib#111]
Any opportunities to collaborate with Google on Kythe?
http://google-opensource.blogspot.com.au/2015/01/kythe-new-approach-to-making-developer.html

Seems like there could at least be a dialog about lessons learnt. Maybe even sharing code down the line.

Anyone here reached out to Google? Or the reverse?

**Comment by sqs** [comment]
Yep, I met with the Kythe team up in Kirkland last August. As much as they wished, they weren't able to share stuff at the time, but now that it's public, we'll reconnect with them. I think there's a lot srclib could learn and use from Kythe, and we're going to dig through their release over the next few days.

**Comment by jokeyrhyme** [comment]
It would be amazing if a shared standard could emerge so that srclib and kythe plugins could work together to some degree.

**Comment by bzz** [comment]
Hi, are there any updates on the topic?

**Comment by sqs** [comment]
Not yet, sorry.

**Comment by fgalassi** [comment]
Any update now ? :)

**Comment by beyang** [comment]
We're finalizing a srclib roadmap proposal for the next 6 months that incorporates many features of the Kythe schema as well as feedback we've gathered from the srclib community and our own use cases. There will be major updates to the srclib schema and toolchain architecture that will make it more general/powerful while maintaining as much simplicity in the schema and API. We've learned a lot from how people are using this in the wild, so thanks to everyone who submitted feedback. We hope to release this sometime in the next 2 weeks and will run it by the community and the Kythe folks. Just need to pin down a few more details and make sure we're addressing everyone's suggestions and ideas. Look forward to releasing that soon!
